src/utils/mailer.js

```
@@ -37,6 +37,13 @@ const createTransport = async () => {
         refreshToken,
         accessToken
       }
+      // Uncomment the following line **only if** you encounter a TLS error during execution.
+      // This disables SSL certificate validation, which can be a security risk.
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)